infinity/custom_items/general/shraken-Harnes-custom_saare.json

```
@@ -1,6 +1,7 @@
 {
-  "ckey" :            "jacking",
-  "character_name" :  "Mammoth Nestor",
+  "ckey" :            "shraken",
+  "character_name" :  "Ethan Harnes",
   "item_path" :       "/obj/item/clothingbag/custom_saare",
   "req_access" :      [ "ACCESS_SECURITY" ]
-  }
\ No newline at end of file
+
+}
```

Пустая строка есть, но она нужна в **самом** конце файла, а не перед ним. Иными словами, пустая строка должна быть после фигурной скобки, а не перед ней.

```suggestion
}

```